packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,152 @@
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context.
+
+| Prop                    | Type                                    | Default     | Description                                                                                                                                        | Required |
+| ----------------------- | --------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `React.ReactText \| React.ReactText[]`  | `undefined` | Renders the label text for the Chip                                                                                                                | ✅       |
+| **icon**                | `Icon`                                  | `undefined` | Displays the Blade Icon component within the Chip                                                                                                  | ❌       |
+| **isDisabled**          | `boolean`                               | `false`     | Disables or enables the Chip                                                                                                                       | ❌       |
+| **intent**              | `'positive' \| 'negative' \| 'neutral'` | `'neutral'` | Sets the ChipGroups's visual variant, it will propagate down to all the Chips                                                                      | ❌       |
+| **value (html native)** | `string`                                | `undefined` | The value of the input field in Chip, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) | ❌       |
+
+## `ChipGroup` API
+
+| Prop                   | Type                                                             | Default     | Description                                                                                                                    | Required |
+| ---------------------- | ---------------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **accessibilityLabel** | `string`                                                         | `undefined` | Accessibility label for the ChipGroup                                                                                          | ✅       |
```

Comment:
Mention that the required behaviour of `accessibilityLabel` is intentional until we introduce a Label component of varying sizes for all our inout elements - this will be in the pipeline.

Reply:
Added.